src/postprocess/chi0.jl

```diff
@@ -180,7 +186,7 @@ is false, only compute excitations inside the provided orbitals.
                 # ∑_{n,m != n} (fn-fm)/(εn-εm) ρnm <ρmn|δV>
                 ρnm = conj(ψnk_real) .* ψmk_real
                 weight = dVol * dot(ρnm, δV)
-                δρk .+= real(ratio .* weight .* ρnm)
+                δρk .+= (n == m ? 1 : 2) * real(ratio .* weight .* ρnm)
```

Comment:
Can you add a comment either here or in the loop above? `for m = n:nbands` is very very easy to confuse with `for m=1:nbands`

Reply:
Also fuse the multiplication